### PR TITLE
Add Radioshack [TKR-434]

### PR DESCRIPTION
**Pull request by idokleinman**
## Description

Add Radioshack (app.radioshack.org):
Mainnet, Polygon, BSC, Avalance (most liquidity)
https://defillama.com/protocol/radioshack

Router contracts:
https://etherscan.io/address/0x3e445e3280C5747a188DB8d0aB7762838A50E4ff
https://polygonscan.com/address/0xaf877420786516fc6692372c209e0056169eebaf
https://bscscan.com/address/0x5bc3ED94919af03279c8613cB807ab667cE5De99
https://snowtrace.io/address/0x763D8D37669bB18064b410e17E3bB46BCF34F487

## Testing instructions

Run Simbot on those 4 chains to verify.
Add token RADIO

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts [hunk 1]
```diff
@@ -106,6 +106,7 @@ export const SELL_SOURCE_FILTER_BY_CHAIN_ID = valueByChainId<SourceFilters>(
             ERC20BridgeSource.CurveV2,
             ERC20BridgeSource.ShibaSwap,
             ERC20BridgeSource.Synapse,
+            ERC20BridgeSource.RadioShack,
```
Comment (kh-chang) [inline packages/asset-swapper/src/utils/market_operation_utils/constants.ts:109]: `BUY_SOURCE_FILTER_BY_CHAIN_ID` should be updated too (line 227)

**Review by kh-chang — CHANGES_REQUESTED** [review]

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts (hunk 1 above)
Comment (idokleinman) [reply packages/asset-swapper/src/utils/market_operation_utils/constants.ts:109]: Ah! Good catch. Ty, ser!

**Review by kh-chang — APPROVED** [review]

**Review by eobbad — APPROVED** [review]
LGTM

**Comment by idokleinman** [comment]
After running Simbot - Not seeing any significant liquidity on this AMM.
Closing this PR without merging.

**Pull request closed without merging**